counterpartylib/lib/api.py

```diff
@@ -764,6 +779,255 @@ def handle_root(args_path):
                 # Not found
                 return flask.Response(None, 404, mimetype='application/json')
 
+        @dispatcher.add_method
+        def sendrawtransaction(tx_hex):
```

Comment:
given the call to `validate.tx_signature`, is this function specific to micropayments? if so, I'd prefix it with `mpc_`?